### ISSUE TITLE
Form group legends not read by screen readers

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#7046]
**Describe the bug**
Our form groups use `<fieldset>` and `<legend>` elements to group fields. However, the `<legend>` is not read out properly by a screen reader because it is not a direct child of the `<fieldset>` element.

The following is not valid:

```html
<fieldset>
  <div>
    <legend>...</legend>
  </div>
  ...
</fieldset>
```

It should be structured like the following:

```html
<fieldset>
  <legend>...</legend>
  ...
</fieldset>
```

**To Reproduce**
1. Go to Settings > Journal > Masthead
2. See error

**What application are you using?**
3.3 pre-release

**Additional information**
Thanks to @benaltair for raising this at https://github.com/pkp/pkp-lib/issues/7006.
